static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -98,6 +105,9 @@ export function OnboardingLayout({
         ...doc.verify(docParams),
       ],
       nextSteps: doc.nextSteps?.(docParams) || [],
+      onPlatformOptionsClick: doc.onPlatformOptionsClick?.(docParams),
+      onProductSelectionClick: doc.onProductSelectionClick?.(docParams),
+      onPageLoad: doc.onPageLoad?.(docParams),
```

Comment:
You don't want to call them at this stage.

```suggestion
      onPlatformOptionsClick: doc.onPlatformOptionsClick,
      onProductSelectionClick: doc.onProductSelectionClick,
      onPageLoad: doc.onPageLoad,
```

I guess we need `onPageLoad` as we need too initially sync the loader settings with the selected products?

Reply:
so if `onLoad` is optional and is useful  to fire an analytic event when the page loads